server/src/main/java/io/crate/execution/engine/collect/sources/InformationSchemaIterables.java

```diff
@@ -298,6 +304,10 @@ public Iterable<ConstraintInfo> constraints() {
         return constraints;
     }
 
+    public Iterable<ConstraintInfo> tableConstraints() {
```

Comment:
Without looking into the implementations, the difference between `constraints` and `tableConstraints` may be unclear. As it's pg related that non-null constraints are not part of `pg_constraint`, I'd suggest to call it so.

```suggestion
    public Iterable<ConstraintInfo> pgConstraints() {
```